view/frontend/web/template/payment/cardknox-form.html

```diff
@@ -115,6 +112,10 @@
             </fieldset>
         </form>
 
+        <!-- ko foreach: getRegion('messages') -->
+        <!-- ko template: getTemplate() --><!-- /ko -->
+        <!--/ko-->
+        
```

Comment:
I don't see where you moved the message box down on the page, is this it?

Reply:
Yes.